code/modules/modular_computers/file_system/news_article.dm

```diff
@@ -14,20 +14,4 @@
 	calculate_size()
 
 
-// NEWS DEFINITIONS BELOW THIS LINE
-
-/datum/computer_file/data/news_article/space/vol_one
-	filename = "SPACE Magazine vol. 1"
-	server_file_path = 'news_articles/space_magazine_1.html'
-
-/datum/computer_file/data/news_article/space/vol_two
-	filename = "SPACE Magazine vol. 2"
-	server_file_path = 'news_articles/space_magazine_2.html'
-
-/datum/computer_file/data/news_article/space/vol_three
-	filename = "SPACE Magazine vol. 3"
-	server_file_path = 'news_articles/space_magazine_3.html'
-
-/datum/computer_file/data/news_article/space/vol_four
-	filename = "SPACE Magazine vol. 4"
-	server_file_path = 'news_articles/space_magazine_4.html'
\ No newline at end of file
+// NEWS DEFINITIONS BELOW THIS LINE
```

Comment:
No idea. If there's going to be no news ever you could just remove the entire program. I was just cleaning up the Bay-specific stuff that's there so people don't get confused.